index.html

```diff
@@ -84,5 +84,224 @@
+        </div>
+
+        <p class="cardpara">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <div class="card1-button">
+          <p class="card1-button-paragraph">html</p>
+          <p class="card1-button-paragraph">css</p>
+          <p class="card1-button-paragraph">javascript</p>
+        </div>
+
+        <a href="#" class="card1-project">See Project</a>
+      </div>
+      <!-- End of Card 2 -->
+
+      <!-- Card 3 -->
+      <div class="card ">
+        <div>
+          <img src="images/part/snapshootportfolio3.png" alt="Tonic" />
+        </div>
+
+        <div class="card1paragraph-text">
+          <h2>Tonic</h2>
+        </div>
+
+        <div class="card1description-text">
+          <p class="canopy marginright">CANOPY</p>
+
+          <img src="images/part/counter.png" alt="counter" class="counterimg marginright" />
+          <p class="imgtext marginright">Back End Dev</p>
+
+          <img src="images/part/counter.png" alt="counter" class="counterimg marginright" />
+          <p class="imgtext marginright">2015</p>
+        </div>
+
+        <p class="cardpara">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <div class="card1-button">
+          <p class="card1-button-paragraph">html</p>
+          <p class="card1-button-paragraph">css</p>
+          <p class="card1-button-paragraph">javascript</p>
+        </div>
+
+        <a href="#" class="card1-project">See Project</a>
+      </div>
+      <!-- End of Card 3 -->
+
+      <!-- Card 4 -->
+      <div class="card ">
+        <div>
+          <img src="images/part/snapshootportfolio4.png" alt="Tonic" />
+        </div>
+
+        <div class="card1paragraph-text">
+          <h2>Multi-Post Stories</h2>
+        </div>
+
+        <div class="card1description-text">
+          <p class="canopy marginright">CANOPY</p>
+
+          <img src="images/part/counter.png" alt="counter" class="counterimg marginright" />
+          <p class="imgtext marginright">Back End Dev</p>
+
+          <img src="images/part/counter.png" alt="counter" class="counterimg marginright" />
+          <p class="imgtext marginright">2015</p>
+        </div>
+
+        <p class="cardpara">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <div class="card1-button">
+          <p class="card1-button-paragraph">html</p>
+          <p class="card1-button-paragraph">css</p>
+          <p class="card1-button-paragraph">javascript</p>
+        </div>
+
+        <a href="#" class="card1-project">See Project</a>
+      </div>
+      <!-- End of Card 4 -->
+    </section>
+
+    <!-- End of Work section -->
+
+    <!-- About me section -->
+    <section class="aboutsection">
+      <div class="aboutparagraph">
+        <h2 class="name">About</h2>
+        <h2 class="name">Myself</h2>
```

Comment:
This should be just one `h2` tag, like `<h2>About myself</h2>`. It only seemed to be wrapping in the design which means that the text should be wrappable if the screen gets so small that it can't contain all the text

Reply:
Correction will be done. thank you